### DISCUSSION TITLE
not an issue

### DISCUSSION THREAD

**Discussion by chief18** [AlaryVanEeckhout/Mega_Man_ZX_Editor#3] [General]
Hello, this is not an issue, I appreciate so much someone working on Mega Man ZX rom hacking, as rom hacks on NDS are usually confined to Pokemon and Mario, I'm trying to learn a bit about MMZX hacking and in general NDS hacking, so Thank You so much.

**Comment by chief18** [comment]
My ultimate and impossible goal would be the one to create a MMZXA sequel, possibly starting from MMZX or MMZXA itself, but I'm very very far from it

**Reply by AlaryVanEeckhout** [reply]
Interesting. I would definitely like to see a sequel to MMZXA. There is allegedly a ["Mega Man ZX 3"](https://megaman.fandom.com/wiki/Mega_Man_ZXC?so=search) that was cancelled, and the title made me wonder what it could have been like. Perhaps there was going to be a "Model C", just like ZX**A** has Model **A**. C made me think of Ciel and how it could be related to stopping model W once and for all, which would end the MMZX series and make it into a trilogy... Well, at least, that was my impression from the little information "Mega Man ZX Cradle" gave away. A fan made sequel doesn't need to be based on the original developer's idea, so I guess you could come up with another story entirely.

As for me, my initial motivation for creating the editor was being able to create a mod of MMZX where you play from the point of view of Giro. Right now, however, understanding how to modify the game and implementing those modifications as editor features is a very interesting process, even if it takes a lot of time. And hopefully, I can help you achieve your goal too.

**Reply by chief18** [reply]
Hello, thanks for moving to discussions, it didn't come to my mind at the moment of writing.
As for the sequel to MMZXA yes, I know there was a project which was started and never completed or even deleted, with ZXC codename, which could be just in facts standing for Ciel.

There is also a fanmade prequel of MMZX, which was made from the ashes of MMX if I'm not wrong.

Anyway I loved Giro character, so to me your idea is very good, my only problem at the moment is that I miss the NDS filesystem knowledge and all, so having this editor just here is incredibly impressive by itself.
First thing I'll do is to read the wiki to gain knowledge at least on MMZX modding

**Reply by AlaryVanEeckhout** [reply]
Discussions, like many other things in a repository, can be toggled on/off. It was off when you wrote the issue, so you couldn't have posted it as a discussion even if you wanted to.

The creator of Mega Man ZX Prequel also started a project named Mega Man ZX Zeta where you can play as Giro, but I think it takes place between ZX Prequel and the start of ZX, unlike my mod idea that would happen within ZX.

Well, actually, the "NDS filesystem knowledge" is pretty much entirely handled by the [ndspy](https://ndspy.readthedocs.io/en/latest/) module. However, the dialogue system seems to be unique to Mega Man ZX/ZXA and ndspy doesn't have anything for graphics editing/viewing so I had to work on those things by myself. The .VX files will have to be decoded manually as well.

**Comment by AlaryVanEeckhout** [comment]
I enabled discussions and moved this here, just to keep it organized.
